addons/event_sale/models/event.py

```diff
@@ -98,8 +108,10 @@ def _default_product_id(self):
         default=_default_product_id)
     registration_ids = fields.One2many('event.registration', 'event_ticket_id', string='Registrations')
     price = fields.Float(string='Price', digits='Product Price')
-    deadline = fields.Date(string="Sales End")
+    start_sale_date = fields.Date(string="Sales Start")
+    end_sale_date = fields.Date(string="Sales End")
     is_expired = fields.Boolean(string='Is Expired', compute='_compute_is_expired')
+    sale_date_availability = fields.Boolean(string='Is Available', compute='_compute_sale_date_availability')
```

Comment:
Maybe rename to sale_available ? As it is not only linked to date, also product.